apache-apisix/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "display_name": "Apache APISIX",
+  "public_title": "Datadog-APISIX Integration",
+  "integration_id": "apache-apisix",
+  "is_public": true,
+  "maintainer": "dev@apisix.apache.org",
+  "manifest_version": "1.0.0",
+  "name": "apache-apisix",
+  "metric_prefix": "apisix.",
+  "metric_to_check": [
+    "apisix.request.counter",
+    "apisix.request.latency",
+    "apisix.upstream.latency",
+    "apisix.apisix.latency",
+    "apisix.ingress.size",
+    "apisix.egress.size"
+  ],
+  "creates_events": false,
+  "short_description": "Datadog-APISIX Integration",
+  "description": "Datadog-APISIX integration using DogStatsD",
+  "guid": "fb69375e-5742-42e6-81cf-8fc19de610ed",
+  "support": "contrib",
+  "supported_os": [
+    "windows",
+    "mac_os",
+    "linux"
+  ],
+  "categories": [
+    "monitoring",
+    "web",
+    "cloud",
+    "log collection",
+    "autodiscovery",
+    "metrics"
+  ],
+  "type": "check",
+  "assets": {
+    "dashboards": {},
+    "configuration": {},
+    "monitors": {},
+    "saved_views": {},
+    "logs": {
+      "service": "apache-apisix"
```

Comment:
Need @rmanalan @jd 's help 😅 Both @bisakhmondal and I prefer keeping the `log collection` category, but we have no idea what happened here.

![image](https://user-images.githubusercontent.com/2106987/145345657-ea33ab31-7936-4084-ace1-6fc8bbf9ed66.png)

Reply:
`log collection` and `logs` section here are for python integration having a logs configuration, so that the agent can collect logs associated with the integration ; it's not the case here so the github action is failing, let's remove `log collection` category and `logs` section in the manifest

Reply:
oh sure! cc @bisakhmondal

Reply:
oh sure! cc @bisakhmondal

Reply:
ACK. Thanks for the suggestion.